org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
In that PR it was "org.wildfly.modcluster". Reverted now. 

However this - "org.wildfly.mod_cluster" - is currently capability used in the config resource. Do we really need 2 separate capabilities?